content/articles/why-you-should-use-wakanda/index.md

```diff
@@ -0,0 +1,55 @@
+### Introduction
+Wakanda is an open-source platform which authorizes the user to easily and quickly create applications that can be utilized on mobile applications and web application using JavaScript. This definition is sweet enough to catch interest but is obvious that developers don’t fall for this line. Wakanda is supported on Microsoft windows, cloud-ready on back-end and Linux. 
+
+### Features of Wakanda JavaScript Framework 
+There are very nice features which make Wakanda JavaScript stand out in this modern day. 
+
+#### Wakanda database
+The Wakanda Database itself stands out for me, especially when it is paired perfectly with the Wakanda Server. It offers the same experience as MongoDb and CouchDb since it uses modern NoSQL with similar portability as SQLite. Wakanda JavaScript framework saves us stress and time since it is portable and ready to use. 
+
+#### Wakanda server
+The Wakanda server runs and hosts the backend applications in the course of production and development. It is a multi-threaded HTTP server which incorporates server-side JavaScript APIs unmasked in REST. Everything needed is already written for the users in JavaScript. It also saves time since the user does not have to go into configuration tasks. The Wakanda server is also portable. It lowers the cost of both configuration and development by using pre-bundled components.
+
+##### JavaScript application server
+The Wakanda JavaScript app server uses the V8 engine which operates on JavaScript runtime. It exhibits a rich synchronous API to hold filesystems, sockets remote databases and workers. It support asynchronous development style.
+
+##### HTTP server
+The Wakanda HTTP server exposes the autogenerated Wakanda REST API. It is also responsible for serving static application files.
+
+##### Data abstraction layer
+The data abstraction layer is essential for data security. It helps in data manipulation through use of a JavaScript API. It exposes an automated REST API to manipulate and access data.
+
+##### Authentication (directory)
+It helps in exposing both the REST API and server side JavaScript API. It is responsible for authentication of management sessions and users.
```

Comment:
What does it mean? Please explain. Consider using descriptive words instead of beginning your paragraph with an "It".